src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
+
+    subtract(num) {
+      this.result -= num;
+    },
+
+    divide(num) {
+      this.result /= num;
+    },
+
+    multiply(num) {
+      this.result *= num;
+    },
+
+    operate(operation, num) {
+      operation.call(this, num);
+
+      return this;
+    },
```

Comment:
The `operate` method is implemented correctly. It takes a callback and a number, applies the operation, and allows method chaining by returning `this`.